### PR TITLE
Docs: revised deployments tutorial

### PR DESCRIPTION
**Pull request by tpdorsey**
### Summary

Updates Deployments tutorial to reflect changes as of 2.1.1.

### Checklist

This pull request is:

- [X] A documentation / typographical error fix
	- No tests or issue needed
- [ ] A short code fix
	- Please reference the related issue by including "closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a bug report issue 
	- Please include tests. One-line fixes without tests will not be accepted unless it's related to the documentation only.
- [ ] A new feature implementation
	- Please reference the related issue by including "closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a feature enhancement issue 
	- Please include tests
    - Please make sure that your QA steps are both thorough and easy to reproduce by somebody with limited knowledge of the feature that you are submitting

**Happy engineering!**

### CODE REVIEW

File: docs/tutorials/deployments.md [hunk 1]
```diff
@@ -30,136 +30,191 @@ That's it. To create flow runs based on the deployment, you need a few more piec
...
+$ mkdir prefect-tutorial
+$ cd prefect-tutorial
+```
+</div>
+
+You may organize your flow scripts and dependencies in any way that suits for team's needs and standards. For this tutorial, we'll keep files within this directory.
 
 ## From flow to deployment
 
 As noted earlier, the first ingredient of a deployment is a flow script. You've seen a few of these already, and perhaps have written a few if you've been following the tutorials.
```
Comment (billpalombi) [inline docs/tutorials/deployments.md:50]:
```suggestion
As noted earlier, the first ingredient of a deployment is a flow script. You've seen a few of these already, and perhaps have written a few, if you've been following the tutorials. 
```

File: docs/tutorials/deployments.md [hunk 2]
```diff
@@ -30,136 +30,191 @@ That's it. To create flow runs based on the deployment, you need a few more piec
...
+## Deployment creation steps
 
 To create a deployment from an existing flow script, there are just a few steps:
 
-1. Use the `prefect deployment build` Prefect CLI command to build a deployment manifest and a deployment YAML file. These describe the files and settings needed to create your deployment.
-1. If you needed to add customizations to the manifest or the deployment YAML file, you would do so now. 
-1. Use the `prefect deployment apply` Prefect CLI command to create the deployment on the API based on the settings in the deployment manifest and deployment YAML.
+1. Use the `prefect deployment build` Prefect CLI command to build a deployment definition YAML file that settings needed to create your deployment and execute flow runs from it. This step also uploads your flow script and any supporting files to storage, if you've specified storage for the deployment.
+1. Optionally, you can edit the deployment YAML file to include additional settings that are not easily specified via CLI flags. 
+1. Use the `prefect deployment apply` Prefect CLI command to create the deployment on the API based on the settings in the deployment YAML file.
```
Comment (billpalombi) [inline docs/tutorials/deployments.md:125]:
```suggestion
1. Use the `prefect deployment apply` Prefect CLI command to create the deployment with the Prefect Orion server based on the settings in the deployment YAML file.
```

File: docs/tutorials/deployments.md [hunk 3]
```diff
@@ -190,14 +246,15 @@ parameter_openapi_schema:
   definitions: null
 ```
 
-## Running Prefect Orion
+Note that the YAML configuration includes the ability to add a description, a default work queue, tags, as schedule, and more.
```
Comment (billpalombi) [inline docs/tutorials/deployments.md:249]:
```suggestion
Note that the YAML configuration includes the ability to add a description, a default work queue, tags, a schedule, and more. 
```

File: docs/tutorials/deployments.md [hunk 4]
```diff
@@ -307,11 +385,11 @@ Note that you can't **Run** the deployment from the UI yet. As mentioned at the
 
 Work queues let you organize flow runs into queues for execution. Agents pick up work from queues and execute the flows.
 
-There is no default global agent, so to orchestrate runs of `leonardo_dicapriflow` you need to configure one. 
+There is no default global agent, so to orchestrate runs of `log_flow` you need to configure one. 
 
 In the Prefect UI, you can create a work queue by selecting the **Work Queues** page, then creating a new work queue. However, you don't need to manually create a work queue because it was created automatically when you created your deployment. If you hadn't created your deployment yet, it would be created when you start your agent. 
 
-In your terminal, run the `prefect agent start` command, passing a `-q test` option that tells it to pull work from the `test` work queue. Remember, we configured the deployment to use this work queue at an earlier step.
+Open a third terminal session, then run the `prefect agent start` command, passing a `-q test` option that tells it to pull work from the `test` work queue. Remember, we configured the deployment to use this work queue at an earlier step.
```
Comment (billpalombi) [inline docs/tutorials/deployments.md:392]:
```suggestion
Open an additional terminal session, then run the `prefect agent start` command, passing a `-q test` option that tells it to pull work from the `test` work queue. Remember, we configured the deployment to use this work queue at an earlier step.
```

**Review by billpalombi — APPROVED** [review]
Minor tweaks, otherwise lgtm.

File: docs/tutorials/deployments.md [hunk 5]
```diff
@@ -30,136 +30,191 @@ That's it. To create flow runs based on the deployment, you need a few more piec
...
 
 ```python hl_lines="14"
+import prefect
 from prefect import flow, task, get_run_logger
 
 @task
-def log_message(name):
+def log_task(name):
     logger = get_run_logger()
     logger.info(f"Hello {name}!")
```
Comment (anna-geller) [inline docs/tutorials/deployments.md:109]:
```suggestion
    logger.info("Hello %s!", name)
```

**Review by anna-geller — APPROVED** [review]
thanks for adding this!

**Pull request merged**